apollos/src/core/components/pullToRefresh/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { Component, PropTypes } from "react"
+import ReactMixin from "react-mixin"
+import { connect } from "react-apollo";
+import ReactPullToRefresh from "react-pull-to-refresh";
+
+const mapStateToProps = (state) => ({
+  isLive: state.live.show,
```

Comment:
This property in the store currently doesn't know whether or not the stream is actually live, just whether to display it if it is live. We should either add another property to the store, or use the actual query.